mne_qt_browser/_pg_figure.py

```diff
@@ -64,6 +63,66 @@ def capture_exceptions():
 name = 'pyqtgraph'
 
 
+# This can be removed when mne==1.0 is released.
+try:
+    from mne.viz.backends._utils import _init_mne_qtapp
+except ModuleNotFoundError:
```

Comment:
```suggestion
except ImportError:
```
because the error I actually get is:
```
>>> from mne.viz.backends._utils import _init_mne_qtapp
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name '_init_mne_qtapp' from 'mne.viz.backends._utils' (/Users/larsoner/python/mne-python/mne/viz/backends/_utils.py)
```